src/cli-ux/README.md

```diff
@@ -0,0 +1,329 @@
+Ervin Howell             Deckow-Crist
+Clementine Bauch         Romaguera-Jacobson
+Patricia Lebsack         Robel-Corkery
+Chelsey Dietrich         Keebler LLC
+Mrs. Dennis Schulist     Considine-Lockman
+Kurtis Weissnat          Johns Group
+Nicholas Runolfsdottir V Abernathy Group
+Glenna Reichert          Yost and Sons
+Clementina DuBuque       Hoeger LLC
+
+$ example-cli users --extended
+Name                     Company            ID
+Leanne Graham            Romaguera-Crona    1
+Ervin Howell             Deckow-Crist       2
+Clementine Bauch         Romaguera-Jacobson 3
+Patricia Lebsack         Robel-Corkery      4
+Chelsey Dietrich         Keebler LLC        5
+Mrs. Dennis Schulist     Considine-Lockman  6
+Kurtis Weissnat          Johns Group        7
+Nicholas Runolfsdottir V Abernathy Group    8
+Glenna Reichert          Yost and Sons      9
+Clementina DuBuque       Hoeger LLC         10
+
+$ example-cli users --columns=name
+Name
+Leanne Graham
+Ervin Howell
+Clementine Bauch
+Patricia Lebsack
+Chelsey Dietrich
+Mrs. Dennis Schulist
+Kurtis Weissnat
+Nicholas Runolfsdottir V
+Glenna Reichert
+Clementina DuBuque
+
+$ example-cli users --filter="company=Group"
+Name                     Company
+Kurtis Weissnat          Johns Group
+Nicholas Runolfsdottir V Abernathy Group
+
+$ example-cli users --sort=company
+Name                     Company
+Nicholas Runolfsdottir V Abernathy Group
+Mrs. Dennis Schulist     Considine-Lockman
+Ervin Howell             Deckow-Crist
+Clementina DuBuque       Hoeger LLC
+Kurtis Weissnat          Johns Group
+Chelsey Dietrich         Keebler LLC
+Patricia Lebsack         Robel-Corkery
+Leanne Graham            Romaguera-Crona
+Clementine Bauch         Romaguera-Jacobson
+Glenna Reichert          Yost and Sons
+```
+
+# CliUx.ux.tree
+
+Generate a tree and display it
+
+```typescript
+let tree = CliUx.ux.tree()
+tree.insert('foo')
+tree.insert('bar')
+
+let subtree = CliUx.ux.tree()
+subtree.insert('qux')
+tree.nodes.bar.insert('baz', subtree)
+
+tree.display()
+```
+
+Outputs:
+```shell
+├─ foo
+└─ bar
+   └─ baz
+      └─ qux
+```
+
+# CliUx.ux.progress
+
+Generate a customizable progress bar and display it
+
+```typescript
+const simpleBar = CliUx.ux.progress()
+simpleBar.start()
+
+const customBar = CliUx.ux.progress({
+                   format: 'PROGRESS | {bar} | {value}/{total} Files',
+                   barCompleteChar: '\u2588',
+                   barIncompleteChar: '\u2591',
+                 })
+customBar.start()
+```
+
+Outputs:
+```shell
+bar1:
+progress [=====================-------------------] 53% | ETA: 1s | 53/100
+bar2:
+PROGRESS | █████████████████████████████░░░░░░░░░░░ | 146/204 Files
+```
+
+To see a more detailed example, run
+```shell script
+$ ts-node examples/progress.ts
+```
+
+This extends [cli-progress](https://www.npmjs.com/package/cli-progress)
+see all of the options and customizations there, which can be passed in with the options object.
```

Comment:
Where is "there"?  Can you clarify?

Reply:
There is the linked reference to cli-progress module.

Reply:
Gotcha.  How about changing this sentence to:

"See the `cli-progress` module for all the options and the customizations that can be passed in with the `options` object."   

(not sure if I got that correct, change as needed.)